Src/IronPython/Runtime/Bytes.cs

```diff
@@ -35,8 +35,9 @@ public class Bytes : IList<byte>, IReadOnlyList<byte>, IEquatable<Bytes>, ICodeF
             _bytes = bytes.ToArray();
         }
 
-        public Bytes([BytesLike, NotNull]ReadOnlyMemory<byte> bytes) {
-            _bytes = bytes.ToArray();
+        public Bytes([BytesLike, NotNull]IBufferProtocol source) {
```

Comment:
If the `ReadOnlyMemory<byte>` overload is gone can we get rid of the `byte[]` overload now?

Reply:
Yes, I intend to do that in the next commit, which will be removal of some obsolete code, like support for `[BytesLike]ReadOnlyMemory` parameters in the binder and the analyzer.